jobs/deploy-znd.groovy

```diff
@@ -282,6 +282,17 @@ def deployToGatewayConfig() {
    }
 }
 
+def createCloudRunTags(){
+   echo("Creating Cloud Run tags.")
+   dir("webapp") {
+      try {
+         exec(["deploy/update_cloud_run_tags.py", VERSION]);
```

Comment:
You need to pass in the list of services being deployed here too.  And also get rid of the try/catch.